fabtests/functional/rdm_stress.c

```diff
@@ -507,6 +520,19 @@ static int rpc_write_req(struct rpc_ctrl *ctrl)
 		goto free;
 	}
 
+	if (fi->domain_attr->mr_mode & FI_MR_ENDPOINT) {
+		ret = fi_mr_bind(ctrl->mr, &ep->fid, 0);
+		if (ret) {
+			FT_ERR("fi_mr_bind failed: %d", ret);
+			goto close;
+		}
+		ret = fi_mr_enable(ctrl->mr);
+		if (ret) {
+			FT_ERR("fi_mr_enable failed: %d", ret);
+			goto close;
+		}
+	}
```

Comment:
Can you move the registration to a new function call, so we can avoid duplicating this?  Just pass in the access flags (FI_REMOTE_READ or FI_REMOTE_WRITE).

Reply:
Will do, thanks.